reqs/test.txt

```diff
@@ -1,2 +1,3 @@
 pytest==6.2.5
 pytest-cov
+apache-airflow[postgres,s3,statsd]==2.2.2
```

Comment:
How is this relevant for this change?

Reply:
Tests don't run without installing this library. It is also installed in reqs/dev.txt but we don't need all the other packages so I just included it test.txt as well

Reply:
That part I understand, but I think it was added even before in a different PR. Otherwise test wouldn't have passed. So I'm just not sure why it is added here.

Reply:
Or I don't understand how the tests were passing before.

Reply:
Should I remove it from the pr?